src/luascript.cpp

```diff
@@ -13685,6 +13686,18 @@ int LuaScriptInterface::luaItemTypeGetWorth(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaItemTypeGetStackSize(lua_State* L)
+{
+	// itemType:getStackSize()
```

Comment:
Suggestion
```
itemType:stackSize(value) // set
itemType:stackSize() // get
```

Reply:
it's used only in shop system
setting stack size should be done only on startup, your change reguest will affect ItemType that is and should be static

Reply:
You're right, I hadn't seen that 😲

Reply:
not a fan of this "revscriptsys" way of getters and setters, make it obvious getAnything, setAnything
anything() is confusing and may result in undefined behaiour if you by any chance pass value or actually pass no value, which could happen, because lua is a dynamic language